examples/player/simid_player.js

```diff
@@ -306,6 +306,12 @@ class SimidPlayer {
       this.simidIframe_.style.position = "absolute";
 
       this.contentVideoElement_.play();
+
+      const nonlinearDuration = document.getElementById('duration').value * 1000;
+      setTimeout(() => {
```

Comment:
no let's not use setTimeout.  It's possible the user pauses playback (like when they go to a different tab or whatever).  Instead let's use currentTime.

So add some code to the currentTime callback that just does a comparison.

If (this.nonlinearStarTime_ - this.vidoeElement_.currentTime > duration) {
  this.stopAd(StopCode.UNSPECIFIED);
}


Side note: I think we need another stop code. NON_LINEAR_DURATION_COMPLETE

https://github.com/InteractiveAdvertisingBureau/SIMID/issues/389

Reply:
Done!